csharp/test/Ice/middleware/AllTests.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) ZeroC, Inc.
+
+#nullable enable
+
+namespace Ice.middleware;
+
+public class AllTests : global::Test.AllTests
+{
+    public static void allTests(global::Test.TestHelper helper)
+    {
+        var output = helper.getWriter();
+        Ice.Communicator communicator = helper.communicator();
+        testMiddlewareExecutionOrder(communicator, output);
+    }
+
+    // Verifies the middleware execute in installation order.
+    private static void testMiddlewareExecutionOrder(Communicator communicator, TextWriter output)
+    {
+        output.Write("testing middleware execution order... ");
+        output.Flush();
+
+        // Arrange
+        var inLog = new List<string>();
+        var outLog = new List<string>();
+
+        var oa = communicator.createObjectAdapter("");
+
+        oa.add(new MyObjectI(), Ice.Util.stringToIdentity("test"));
```

Comment:
Seems simpler to use `new Identity { name = "test" }` in new code.

Reply:
fixed